packages/gestalt-design-tokens/tokens/vr-theme-web-mapping/comp-web-color-dark.json

```diff
@@ -274,6 +274,27 @@
           }
         }
       },
+      "iconbutton": {
```

Comment:
this needs it's counterpart in the light file.

basically all changes in light classic need to be also in dark classic and light+dark VR

Check the other tokens, they are in all files.

Reply:
Fixed, thought dark mode would fall back into light mode and vr fall back into classic hehe